static/scripts/sc-blog.js

```diff
@@ -1,29 +1,74 @@
 const stripHtml = require('string-strip-html');
 
+let production;
+let linksArray;
+
 function fetchContent() {
+	let finalHtml = '';
 	$('.sc-blog .spinner').show();
 	$('.sc-blog .placeholder').hide();
 
-	$.ajax({
-		url: '/ghost/startseite-hpi-schul-cloud',
-		type: 'GET',
-		dataType: 'json',
-		contentType: 'application/json',
-		timeout: 8000,
-	})
-		.done((result) => {
-			$('.sc-blog .title').text(result.pages[0].title);
-			result.pages[0].html = result.pages[0].html
-				.replace(/<td>x<[/]td>/g, '<td><i class="fa fa-check"></i></td>');
-			$('.sc-blog .content').html(stripHtml(result.pages[0].html, { onlyStripTags: ['script', 'style'] }));
+	const changePage = () => {
+		$('.sc-blog .loading').remove();
+		finalHtml = finalHtml
+			.replace(/<td>x<[/]td>/g, '<td><i class="fa fa-check"></i></td>');
+		$('.sc-blog .content').html(stripHtml(finalHtml,
+			{ onlyStripTags: ['script', 'style'] }));
+		$('.sc-blog .content').css('opacity', '1');
+	};
+
+	const promiseFunc = new Promise((resolve) => {
+		$.ajax({
+			url: '/ghost/landing-page/',
+			type: 'GET',
+			dataType: 'json',
+			contentType: 'application/json',
+			timeout: 8000,
 		})
-		.fail(() => {
-			$('.sc-blog .spinner').hide();
-			$('.sc-blog .placeholder').show();
+			.then((result) => {
+				const regexHash = /\#{1}[a-zA-Z--]+/g;
+				const regexAsterisk = /\~{1}[a-zA-Z--]+/g;
+				if (production) {
+					linksArray = result.pages[0].html.match(regexHash);
+				} else {
+					linksArray = result.pages[0].html.match(regexAsterisk);
+				}
+				resolve();
+			})
+			.fail((error) => {
+				console.log(error);
```

Comment:
Don't show the whole error (maybe even none?). You should see the button and error text if it failed to load.

Reply:
I've added h2 with error text, if You want none error, let me know